jianna-relaxr-blog.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+    	<meta charset="utf-8">
+    	<meta name="description" content="Get piece of mind with a single tap">
+    	<meta name="keywords" content="one, two, three">
+
+		<title>Relaxr Blog</title>
+
+		<!-- external CSS link -->
+		<link rel="stylesheet" href="css/normalize.css"> 
+		<rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/jianna-relaxr-blog.css">
+
+		<link href="https://www.google.com/fonts#UsePlace:use/Collection:Open+Sans" rel="stylesheet">
+
+	</head>
+	<body>
+		<div id="container">
+			<header>
+				<h1>Relaxr</h1>
+				<nav class="menu">
+					<a href="jianna-landingpage.html">About</a>
+					<a href="#">FAQ</a>
+					<a href="#">Team</a>
+					<a href="#">Contact Us</a>
+					<a href="jianna-relaxr-blog.html">Blog</a>
+				</nav>	
+			</header>
+			
+			<div id="body">
```

Comment:
@saraqkr I would stay away from naming id's or classes as "body" because we already have a core HTML element called body, this can possibly cause a conflict in your CSS. Next time you can use "body-container" or "parent-container"